### PR TITLE
feat(Loading): add experimental loading spinner

### LINKED ISSUE #1643: Experimental Component: Inline Loading
Related:

https://github.com/IBM/carbon-components/issues/1193

https://github.com/IBM/carbon-components/issues/1259

https://github.com/IBM/carbon-components/issues/1299

### PR DESCRIPTION
**Pull request by emyarod**
Closes IBM/carbon-components-react#1641
Closes #1643

This PR will add the new experimental loading spinner changes

### CODE REVIEW

File: src/components/Loading/Loading.js [hunk 1]
```diff
@@ -52,7 +53,21 @@ export default class Loading extends React.Component {
         aria-live={active ? 'assertive' : 'off'}
         className={loadingClasses}>
         <svg className={`${prefix}--loading__svg`} viewBox="-75 -75 150 150">
-          <circle cx="0" cy="0" r="37.5" />
+          <title>Loading</title>
+          {componentsX && small ? (
```
Comment (joshblack) [inline src/components/Loading/Loading.js]: As a style convention, do we prefer the ternary pattern or `&&` chain pattern, like:

```js
{componentsX && small && (
  <circle />
)}
```

Comment (asudoh) [reply src/components/Loading/Loading.js]: While I prefer chaining personally, I think this project has accept either approach.

**Comment by joshblack** [comment]
_PR Reviews 1/11_

- [ ] Needs visual PR for sign-off

**Comment by emyarod** [comment]
the Netlify link should now allow reviewer to view the experimental component thanks to https://github.com/IBM/carbon-components-react/pull/1711

**Comment by lovemecomputer** [comment]
looks good, though i feel like the spin speed for the large should be slowed down… it feels jarring/makes me nervous to look at hehe. 

more of a question for getting a value from @IBM/carbon-designers — i think the loading speed at large size feels right with: 

```
.bx--loading {
  animation-duration: 690ms;
  …
}
```

**Comment by emyarod** [comment]
@aagonzales sure these style rule changes will need to be made in the vanilla repo. I've opened a PR to address it: https://github.com/IBM/carbon-components/pull/1633

**Comment by lovemecomputer** [comment]
thanks, @emyarod ! adding my detail to your appropriate PR

**Comment by aagonzales** [comment]
oh oops didn't realize this was for react but yes that would be great! And agree with @lovemecomputer that the speed could be slowed down a little.

**Comment by asudoh** [comment]
@emyarod Given the vanilla style change seems in [`9.66.4`](https://github.com/IBM/carbon-components/pull/1633#issuecomment-454599393), do you want to upgrade the dev dependency? Thanks!

**Comment by emyarod** [comment]
yes @asudoh I have updated the dependency version

**Pull request merged**